packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js

```diff
@@ -66,6 +66,12 @@ class SeleniumDockerService {
 
     await exec(`TERRA_SELENIUM_DOCKER_VERSION=${this.version} docker stack deploy -c ${composeFilePath} wdio`);
 
+    // There are occasions where the docker stack deployment is not completely ready when running back-to-back wdio test runs.
+    // As a result, the test run is unable to start and receives the 'Error forwarding the new session cannot find : Capabilities {browserName: chrome}' error.
+    // This waits for 5 seconds and ensure the services and network are created and ready.
+    await this.wait(5000);
+    await this.waitForServiceCreation();
+    await this.waitForNetworkCreation();
```

Comment:
Were these two service/network waits tested with and without the time wait above? Are all three waits necessary to get reliable test runs or is the 5 second wait enough? These are probably fine to add either way, but I am curious if they play a role in increasing the reliability of avoiding the session error or if the timed wait is the main contributing factor

Reply:
The services and network get created fairly quickly...less than a second from my testing. Unfortunately, just checking the services and network without the wait still gives the error. Adding just the wait alone gives a reliable test run but I thought it doesn't hurt to also verify that the services and network are created. I looked at the [docker stack deploy doc](https://docs.docker.com/engine/reference/commandline/stack_deploy/#examples) but don't see what else I could check other than ensuring that the services and network are created.